docs/pages/ds-video-port_L3_Low-Level_TestSpecification.md

```diff
@@ -0,0 +1,43 @@
+# Device Settings Video Port L3 Low Level Test Specification and Procedure Documentation
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [References](#references)
+- [Level 3 Test cases High Level Overview](#level-3-test-cases-high-level-overview)
+
+## Overview
+
+This document describes the L3 Low Level Test Specification and Procedure Documentation for the Device Settings Video Port module.
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL` \- Hardware Abstraction Layer, may include some common components
+- `UT`  \- Unit Test(s)
+- `OEM` \- Original Equipment Manufacture
+- `SoC` \- System on a Chip
+- `HDMI`\- High-Definition Multimedia Interface
+- `HDCP`\- High-bandwidth Digital Content Protection
+- `HDR` \- High Dynamic Range
+- `HLG` \- Hybrid Log-Gamma
+- `SDR` \- Standard Dynamic Range
+
+### References
+
+- `High Level Test Specification` - [dsVideoPort High Level TestSpec](ds-video-port_High-Level_TestSpec.md)
+- `Interface header` - [dsVideoPort HAL header](https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsVideoPort.h)
+
+## Level 3 Test Cases High Level Overview
+
+|#|TestName|Description|
+|-|--------|-----------|
+|1|Verify the Video content Formats with callbacks|Play the pre-defined streams different video content format(`HDR`,`HLG`,DolbyVision,..) and check the callbacks is triggered when the video content format changes|
+|2|Check DisplayConnected|Verify the Display by enabling and disable port|
+|3|Select the `HDCP` Version |Select the Supported `HDCP` version and verify|
```

Comment:
not sure what this means? Which function in dsVideoPort does this today? 

I think your thinking about extended functionality rather than major goals... Will you not require external hardware that supports earlier versions of HDMI to support this testing?